.env.local

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_PROXY_URL=www.google.com
```

Comment:
.env.local should not be checked in to source control. I'm also not totally sure it'll be picked up right now since I think we need some special setup to use the `.env` files

Reply:
yeah I was being dumb, we're not using next at all here anyway